simple_ldap_user/simple_ldap_user.admin.inc

```diff
@@ -295,10 +295,18 @@ function simple_ldap_user_import($form, &$form_state) {
   asort($users);
 
   // Only selects new users
-  $drupal_users = entity_load('user');
+  
   $new_names = $users;
-  foreach ($drupal_users as $user) {
-      unset($new_names[strtoupper($user->name)]);
+  $sql="select name from {users} where name= :name";
+  foreach($new_names as $item){
```

Comment:
Drupal coding standards require control structures to have a space between the keyword and parenthesis, then a space between the parenthesis and the opening brace.  Please adjust this.